### PR TITLE
README.md edited, project structure improved and absolute paths added

### PR DESCRIPTION
**Pull request by Juan-Cutiva**
Add absolute paths using a vite configuration, now use absolute paths in .tsx files.
Add a new dependency ("@types/node"), this dependency fixes the error in vite.config.ts when importing the "path"
Fixed bugs in tsconfig.json and tsconfig.node.json

### CODE REVIEW

**Comment by aflopezbec** [comment]
@Juan-Cutiva please add a description in the pull request. The description is very important for show the context to other teammates.

File: src/sections/main.tsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import { StrictMode } from 'react'
 import { createRoot } from 'react-dom/client'
-import './index.css'
-import App from './App.tsx'
+import '../css/index.css'
+import App from '../components/App.tsx'
```
Comment (aflopezbec) [inline src/sections/main.tsx]: we need to use absolute imports https://dev.to/smitterhane/avoid-relative-path-import-hell-in-react-36in

File: src/components/App.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import reactLogo from '/assets/react.svg'
+import typescriptLogo from '/assets/typescript.svg'
+import '../css/App.css'
```
Comment (aflopezbec) [inline src/components/App.tsx]: absolute imports, please

**Review by aflopezbec — CHANGES_REQUESTED** [review]

File: src/sections/main.tsx (hunk 1 above)
Comment (Juan-Cutiva) [reply src/sections/main.tsx]: This is fixed

File: src/components/App.tsx (hunk 2 above)
Comment (Juan-Cutiva) [reply src/components/App.tsx]: This is fixed

**Pull request closed without merging**